scripts/lib/qa_crowbarsetup-help.sh

```diff
@@ -12,6 +12,9 @@ function qacrowbarsetup_help
         Option to set variable MTU size or select Jumbo Frames for Admin and Storage nodes. 1500 is used if not set.
     want_raidtype (default='raid1')
         The type of RAID to create.
+    want_batch_dir (default='/root')
```

Comment:
> That would need more rework, not covered by CI, so prone to break things.

Sorry, I don't understand - why isn't it as simple as adding

        export scenariodir=$scenariodir ;

to the creation of `$mkcconf`?  How would this risk breaking anything?

> And I need it for the mkphyscloud case where there is no mkcloud involved anyway.

Yes but if this new feature can be easily supported for both physical and virtual clouds then why not do it?  Virtual clouds is a far more common use case than physical.